Demos/VaporDemo/Sources/App/routes.swift

```diff
@@ -1,11 +1,31 @@
 import Vapor
 import MeiliSearch
 
+private struct Movie: Codable, Equatable {
+
+    let id: Int
+    let title: String
+    let comment: String?
+
+    enum CodingKeys: String, CodingKey {
+        case id
+        case title
+        case comment
+    }
+
+    init(id: Int, title: String, comment: String? = nil) {
+        self.id = id
+        self.title = title
+        self.comment = comment
+    }
+
+}
+
 func routes(_ app: Application) throws {
 
-    let client = try! MeiliSearch(Config(hostURL: "http://localhost:7700"))
+    let client = try! MeiliSearch(Config.default(apiKey: "masterKey"))
 
-    // 127.0.0.1:8080/index?uid=Movies
+    // 127.0.0.1:8080/index?uid=books_test
```

Comment:
```suggestion
    // 127.0.0.1:7700/indexes/books_test
```

Reply:
The port should be `8080` for Vapor test. We are testing the vapor example, not meilisearch.